docs/source/ci-cd.md

```diff
@@ -278,6 +279,26 @@ pipeline {
   }
 }
 ```
+## Gitlab
+Since there isn't any official Docker image for Rover, we can use the `node:lts-alpine3.17` as a base image. All you need to do is fetch the source via cURL, add the executable to the PATH variable, then publish your subgraphs. 
```

Comment:
I don't think I would recommend `node` as the base image here because Rover does not require `node` to run! Additionally `lts-alpine` isn't a great choice either because you lose access to commands like `rover supergraph compose` which are not compiled for `alpine`. I think if you change the base image to `ubuntu` or `debian:stable-slim` this would be a bit better. What do you think?

Reply:
Indeed, we were running a prep script for our case which requires nodejs to generate graphs, so it's a leftover.
Replacing with `debian:stable-slim`
Thanks!